components/drawers/drawers.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Shreyas c. Soni">
+    <!-- <link rel="stylesheet" href="https://ripple-ui.netlify.app/css/components.css" /> -->
```

Comment:
Suggestion: Can remove unnecessary comments.

Reply:
Thanks I will remove that once all my components CSS is in dev